dotnet/src/SemanticKernel.Abstractions/Functions/IKernelPluginExtensions.cs

```diff
@@ -112,7 +112,7 @@ public static IList<KernelFunctionMetadata> GetFunctionsMetadata(this IEnumerabl
         {
             foreach (KernelFunction function in plugin)
             {
-                metadata.Add(new KernelFunctionMetadata(function.Metadata) { PluginName = plugin.Name });
+                metadata.Add(function.Metadata);
```

Comment:
Shouldn't the metadata we add here contain the plugin name?

Reply:
Will revert this change